fhttp/httprunner.go

```diff
@@ -40,10 +40,12 @@ type HTTPRunnerResults struct {
 	sizes       *stats.Histogram
 	headerSizes *stats.Histogram
 	// exported result
-	Sizes       *stats.HistogramData
-	HeaderSizes *stats.HistogramData
-	URL         string
-	SocketCount int
+	Sizes              *stats.HistogramData
+	HeaderSizes        *stats.HistogramData
+	URL                string
+	SentRequestSize    int
```

Comment:
add comments above the new fields and that would also break up the auto indent into multiple chunks making the diff smaller/easier to review

Reply:
Sure

Reply:
ping :)

Reply:
I was waiting for your comments :D.